cloudinary/api.py

```diff
@@ -93,6 +93,25 @@ def resources_by_ids(public_ids, **options):
     return call_api("get", uri, params, **options)
 
 
+def resources_by_asset_ids(asset_ids, **options):
+    """Retrieves the resources (assets) indicated in the asset IDs.
+    This method does not return deleted assets even if they have been backed up.
+
+    See: `Get resources by context API reference
+    <https://cloudinary.com/documentation/admin_api#get_resources>`_
+
+    :param asset_ids:   The requested asset IDs.
+    :type asset_ids:    str
```

Comment:
Please change it to:
`    :type asset_ids:    list[str]`

Reply:
Updated